ga_resources/tasks.py

```diff
@@ -7,7 +7,7 @@
 from django.utils.timezone import utc
 
 
-@task(ignore_result=True)
+#@task(ignore_result=True)
 def refresh_resource(pk):
```

Comment:
> This module is not needed since we are not using GeoAnalytics (GA)

Can we discuss removing this from the codebase during today's session?
The upgrade path to Django 1.8.9 flags elements of both of the ga_ apps.